parse-semver/main.py

```diff
@@ -44,7 +44,7 @@ def main():
     print("::endgroup::")
 
     set_output("version", parsed_version.public)
-    set_output("next-minor-version", next_parsed_version.public)
+    set_output("next-minor-alpha-version", next_parsed_version.public)
```

Comment:
this is what downstream workflows expect as the output name: https://github.com/dbt-labs/actions/blob/main/.github/workflows/cut-release-branch.yml#L227